### DISCUSSION TITLE
Couldn't find any Elasticsearch data

### DISCUSSION THREAD

**Discussion by pilgrimage80** [chaoss/grimoirelab#638] [Q&A]
I filled project.json and setup.cfg according to the steps in the document to pull Gitlab data, but I get the following error with localhost:5601, what could be the reason? 

Message 👍 
No default index pattern. You must select or create one to continue.

[affiliations]
[all_enriched]
[all_enriched_tickets
[all_onion]
[demographics]

Couldn't find any Elasticsearch data
You'll need to index some data into Elasticsearch before you can create an index pattern.

**Comment by sduenas** [comment]
This is an issue related to Elastic Search/Kibana. Please read the documentation of Kibana about how to create dashboards and visualizations.

https://www.elastic.co/guide/en/kibana/7.17/kibana-concepts-analysts.html
https://www.elastic.co/guide/en/kibana/7.17/index-patterns.html
